dbt_copilot_helper/commands/dns.py

```diff
@@ -749,5 +756,301 @@ def assign(app, domain_profile, project_profile, svc, env):
         return
 
 
+def update_rule(delete, listener, conditions, rule_arn, elb_client, acm_client, cdn_domain):
+    # Update Rule
+    response = elb_client.modify_rule(RuleArn=rule_arn, Conditions=conditions)
+
+    # Create certificate if not already present.
+    base_domain = get_base_domain([cdn_domain])
+    domain_profile = AVAILABLE_DOMAINS[base_domain]
+    domain_session = get_aws_session_or_abort(domain_profile)
+    r53_client = domain_session.client("route53")
+    cert_arn = create_required_zones_and_certs(
+        r53_client,
+        acm_client,
+        cdn_domain,
+        base_domain,
+    )
+
+    # Attach/Remove certificate from LB.
+    if delete:
+        response = elb_client.remove_listener_certificates(
+            ListenerArn=listener["ListenerArn"],
+            Certificates=[
+                {
+                    "CertificateArn": cert_arn,
+                },
+            ],
+        )
+    else:
+        # breakpoint()
+        response = elb_client.add_listener_certificates(
```

Comment:
I don't think `response` it used anywhere...
```suggestion
        elb_client.add_listener_certificates(
```

Reply:
updated with check